app/controllers/application_controller.rb

```diff
@@ -1,5 +1,12 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    @current_user = User.first
+  protect_from_forgery with: :exception
+
+  before_action :update_allowed_parameters, if: :devise_controller?
+
```

Comment:
- [ ] When I run your app. It shows me a blank page. That is not quite right. But we can fix it by requiring authentication before action. To make it happen kindly accept my suggestion:

```suggestion
  before_action :authenticate_user!
  
```

Reply:
Thanks 
I have added it to it